src/chart/Treemap.tsx

```diff
@@ -419,8 +421,30 @@ export class Treemap extends PureComponent<Props, State> {
   };
 
   handleClick(node: TreemapNode) {
-    const { onClick, type } = this.props;
-    if (type === 'nest' && node.children) {
+    const { onClick, type, children } = this.props;
+    const tooltipItem = findChildByType(children, Tooltip);
+    const tooltipTrigger = tooltipItem.props.trigger;
+    const { activeNode } = this.state;
+    const isNestedParentNode = type === 'nest' && node.children;
+
+    if (tooltipTrigger === 'click') {
```

Comment:
How much of this is copy-paste from `generateCategoricalChart`? Just wondering.

Reply:
I just hacked this to get it it to work, no references from anywhere except Treemap